config/bpf-bytecode/tc-egress.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: bpfd.dev/v1alpha1
+kind: TcProgram
+metadata:
+  labels:
+    app.kubernetes.io/name: blixt-tc-egress
+  name: tc-egress
+spec:
+  bpffunctionname: tc_egress
+  # Select all nodes
+  nodeselector: {}
+  interfaceselector:
+    primarynodeinterface: true
+  priority: 0
+  direction: egress
+  bytecode:
+    image:
+      url: quay.io/bpfd-bytecode/blixt-tc-egress:latest
```

Comment:
Should we put a TODO here to get this image pushed to the same place we push our other images?

(note that we have #106 to deal with too, I hadn't gotten around to it yet)